webapp-django/crashstats/base/ga.py

```diff
@@ -0,0 +1,135 @@
+"""
+Sending Google Analytics events from the webapp.
+
+See https://developers.google.com/analytics/devguides/collection/protocol/\
+v1/devguide#page
+"""
+
+import logging
+import urlparse
+import uuid
+
+from raven.transport.threaded_requests import ThreadedRequestsHTTPTransport
+
+from django.contrib.sites.requests import RequestSite
+from django.conf import settings
+
+logger = logging.getLogger('crashstats:ga')
+
+# Uncomment for local debugging. This will print all logging on stdout.
+# logger.setLevel(logging.DEBUG)
+# import sys
+# logger.addHandler(logging.StreamHandler(sys.stdout))
+
+
+def track_api_pageview(
+    request,
+    page_title=None,
+    data_source='api',
+    client_id=None,
+    async=True,
```

Comment:
I thought I couldn't mock the async nature and since I know that tests would always need to be "eager" I thought this could come in handy. 
But I'll remove it now. It's silly to have code checked in until it's needed.